### ISSUE TITLE
get_players not works

### ISSUE THREAD

**Issue by Karaulov** [alliedmodders/amxmodx#790]
I can't see some players with this function!!!

For example user "CS-BEST SERVER CHECKER ＃4204" invisible for this function.

**Comment by Giferns** [comment]
maybe he is still connecting to server? get_players() have
`"i" - include connecting clients`

**Comment by Karaulov** [comment]
This problem in default plugin admincmd.sma (command amx_who)

**Comment by CrazyHackGUT** [comment]
In this plugin, flag `i` doesn't used. Player is fully connected?

**Comment by Karaulov** [comment]
> In this plugin, flag `i` doesn't used.

Yes, it problem. (I can't get user id for ban this player, because him not connected)

**Comment by OciXCrom** [comment]
Connecting clients aren't displayed with `amx_who` - you usually don't need to ban anyone before he even connects. Anyways, you can see his id by typing `vote` in the console and ban him using regular CS commands.

**Comment by Karaulov** [comment]
> Connecting clients aren't displayed with `amx_who` - you usually don't need to ban anyone before he even connects. Anyways, you can see his id by typing `vote` in the console and ban him using regular CS commands.

Fake clients invisible by amx_who command, and why amx_who not displaying IP address ??

**Comment by afwn90cj93201nixr2e1re** [comment]
You can't get info about not connected players, and thats expected.

**Comment by Karaulov** [comment]
@afwn90cj93201nixr2e1re i see these players in scoreboard. Why i can kick this players?

**Comment by afwn90cj93201nixr2e1re** [comment]
It's hlds issue, not related to amxmodx :/

**Comment by Karaulov** [comment]
> It's hlds issue, not related to amxmodx :/

No.

**Comment by afwn90cj93201nixr2e1re** [comment]
Ты идиот? Что ты несешь?
